pinc/comment_inclusions.inc

```diff
@@ -41,7 +44,14 @@ function parse_project_comments($comments) {
 			$i++;
 		}
 	}
-	return $comments;
+
+	if ($project->unsafe_comment) {
+		return ($comments);
```

Comment:
Spacing (tab not spaces), also below.

Reply:
These are tabs, unless I'm mistaken as to what you refer.